.github/workflows/deploy-stacks.yml

```diff
@@ -0,0 +1,117 @@
+name: deploy-stacks.yml
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  create-env-change-sets-and-deploy:
+    permissions:
+      id-token: write
+      contents: read
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node and install cdk cli
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Install dependencies and build project
+        run: |
+          npm install
+          npm install -g aws-cdk
```

Comment:
Do we need to specify the version just to be sure not to break the compatibility with the current code-base?

Reply:
I would rather have it break to make sure we always use the most recent LTS version.

Reply:
We are already lagging behind on cdk version.